integrations/oci/metrics/cdi/src/main/java/io/helidon/integrations/oci/metrics/cdi/OciMetricsConfigProducer.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2024 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.integrations.oci.metrics.cdi;
+
+import javax.annotation.Priority;
+import javax.enterprise.inject.Produces;
+
+import io.helidon.config.Config;
+
+@Priority(0)
+class OciMetricsConfigProducer {
+
+    @Produces
+    @OciMetrics
+    public Config config(Config config) {
```

Comment:
This could be `private static` if you wanted; then no instance of `OciMetricsConfigProducer` will have to be created.

Reply:
I do not seem to be able to keep in my head the rules/conventions for visibility to CDI vs. Java declarations. I'll make this change.

Reply:
To be fair: it's just another method. If you wanted to surface it, leave it as `public` for sure. But often producer methods are to be used solely by the container. If that's the case as I strongly guess it is, then I personally prefer to limit visibility wherever I can.